### PR TITLE
Feat: 1830 Update library Template

### PR DESCRIPTION
**Pull request by abdul-akibu**
## What it does
- This PR updates the library template in lsc so it can build correctly with ng-packgr.

### CODE REVIEW

**Comment by abdul-akibu** [comment]
still need to finish up some testing with using lsc build lib instead of the npm run command. Should finish up tomorrow.

File: templates/library-package/README.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+* If you need to load an image in your component:
+    1. Add a variable to your component as follows. E.g.: `myImg = require('@labshare/ngx-labshare-base/src/assets/my-image.png')`
+    2. Inject into template: `<img [src]="myImg" />`
+    * Please note that the require path above **must** be referenced from `@labshare/ngx-labshare-base/src/assets` to ensure consistent loading behavior.
+
+* For more information, go to @labshare/shell-ui.
+
+## Contents
+
+- **scripts** - Any sctipts needed for your library.
```
Comment (JamilOmar) [inline templates/library-package/README.md]: there is a typo here

File: templates/library-package/package.json [hunk 2]
```diff
@@ -3,8 +3,13 @@
   "version": "1.0.0",
```
Comment (JamilOmar) [inline templates/library-package/package.json]: start with version 0.0.0 , if we start with version 1.0.0 the semantic release will break

**Pull request merged**